lambda/format.js

```diff
@@ -27,7 +27,7 @@ exports.default = async (params) => {
       let returnedImage;
       if (conversionFormat.extension !== extension) {
         const resizedReformatted = `${originalResized.split('.')[0]}.${conversionFormat.extension}`;
-        const reformatProcess = childProcess.spawnSync(appPath, [originalResized, resizedReformatted]);
+        const reformatProcess = childProcess.spawnSync(appPath, [originalResized, '-interlace', 'plane', resizedReformatted]);
```

Comment:
Won't this impact all image formats (png and gif included)? Shouldn't it be applied only to jpegs?

Reply:
I don't think a gif or png would make it through the conditional for this block

Reply:
Which one? By following the code to me it looks like the above is called for every file type.